app/helpers/data_getter.py

```diff
@@ -216,16 +216,15 @@ def get_tracks_by_event_id():
         return Track.query.filter_by(event_id=get_event_id())
 
     @staticmethod
-    def get_sessions(event_id, state='accepted'):
```

Comment:
The `state` parameter is there for a reason. It allows us to re-use this method anywhere else we want. Maybe in someplace we want only rejected sessions. At that time we want to reuse this same method by calling `get_sessions(1, 'rejected')`. 

Please don't just blindly remove things like this without considering why it is there in the first place.